packages/react-native-bpk-component-calendar/src/NativeCalendar.ios.js

```diff
@@ -86,10 +97,10 @@ class BpkCalendar extends Component<Props, {}> {
     return (
       <RCTBPKCalendar
         ref={this.calendarRef}
-        minDate={minDate}
-        maxDate={maxDate}
+        minDate={parseDateToNative(minDate)}
+        maxDate={parseDateToNative(maxDate)}
         onDateSelection={onChangeSelectedDates}
-        selectedDates={selectedDates}
+        selectedDates={selectedDates.map(parseDateToNative)}
```

Comment:
This was a nightmare issue to fix. Essentially something appears to have changed on the RN iOS side that means dates sent over the bridge weren't making it over properly, so it was failing to parse and causing red box errors.

I never figured out what exactly changed internally to make this happen, but I fixed it by taking inspiration from the Android version of this component and cast everything to a timestamp before sending it over the bridge, which makes it work.